lib/constants.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Are these for internal use or external use?

Reply:
Intended for external use if desired

Reply:
Gotcha. I'm not sure I see a _ton_ of value in including it, but not opposed. If you expect clients to use this, we could wrap it in `Object.freeze` to prevent mutation: 

```js
> const a = Object.freeze({ key: 'bloop' });
undefined
> a.key
'bloop'
> a.key = 'bleep';
'bleep'
> a.key
'bloop'
```